### PR TITLE
60 simple hash

### LINKED ISSUE #60: Add simple hash variant
Taking last 20 bytes of file as hash

### PR DESCRIPTION
**Pull request by idruzhitskiy**
Closes #60

### CODE REVIEW

File: src/RemoteFileCopy/FilesFinding/FilesFinder.cs [hunk 1]
```diff
@@ -17,6 +17,8 @@ namespace RemoteFileCopy.FilesFinding
 {
     public class FilesFinder
     {
+        private const string ShaHashFunction = "$(sha1sum < $f | awk '{{ print $1 }}')";
+        private const string SimpleHashFunction = "$(head -c 20 $f | hexdump -e '16/1 \"%02x\"')";
```
Comment (ikopylov) [inline src/RemoteFileCopy/FilesFinding/FilesFinder.cs:21]: For simple mode it is better to compare file size + hash of last 1Mb of file + probably, hash of the first 4kb of file

File: src/RemoteFileCopy/FilesFinding/FilesFinder.cs [hunk 2]
```diff
@@ -12,20 +12,26 @@ namespace RemoteFileCopy.FilesFinding
 {
     public class FilesFinder
     {
+        private const string ShaHashFunction = "$(sha1sum < $f | awk '{{ print $1 }}')";
+        private const string SimpleHashFunction = "$({ head -c 4096 $f ; tail -c 1073741824 $f ; } | hexdump -e '16/1 \"%02x\"')";
```
Comment (ikopylov) [inline src/RemoteFileCopy/FilesFinding/FilesFinder.cs:16]: In this case file sizes should also be compared. Probably, file size can be include into this function

File: src/RemoteFileCopy/FilesFinding/FilesFinder.cs [hunk 3]
```diff
@@ -47,13 +53,23 @@ public FilesFinder(SshWrapper sshWrapper, ILogger<FilesFinder> logger)
...
+        internal static string GetBashHereDoc(string path, string hashFunction)
         {
             return $@"'EOF'
 if [ -d {path} ]
 then 
     for f in $(find {path} -type f -print) ; do
-        hash=$(sha1sum < $f | awk '{{ print $1 }}')
+        hash={hashFunction}
         size=$(stat -c%s $f)
         echo f$f l$size c$hash
```
Comment (ikopylov) [inline src/RemoteFileCopy/FilesFinding/FilesFinder.cs:74]: Is this `size` used in comparison? If yes, then ignore my comment on `SimpleHashFunction`

Comment (idruzhitskiy) [reply src/RemoteFileCopy/FilesFinding/FilesFinder.cs:74]: Yes

File: src/RemoteFileCopy/FilesFinding/FilesFinder.cs (hunk 2 above)
Comment (idruzhitskiy) [reply src/RemoteFileCopy/FilesFinding/FilesFinder.cs:16]: Size is always compared, no matter what function is being used

**Comment by ikopylov** [comment]
Build was broken. Please, fix it

**Review by ikopylov — APPROVED** [review]
LGTM

**Pull request merged**